app/controllers/tasks_controller.rb

```diff
@@ -13,6 +13,12 @@ def index
         render json: Task.all
     end
 
+    def destroy
+        task = Task.find(params[:id])
```

Comment:
What would happen if this `task` couldn't be found?

Reply:
+1

Reply:
If the task won't be found it will raise an ActiveRecord::RecordNotFound error. I might need help here how to manage this.

Reply:
You could create an ExceptionHandler that will rescue those errors from ActiveRecord. 
example: `rescue_from ActiveRecord::RecordNotFound`